### ISSUE TITLE
why my  trained yolov5s weights is so big ?about 54.5MB

### ISSUE THREAD

**Issue by jiangjiaxiaotianxia** [ultralytics/yolov5#6417]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I use the 6.0 vision yolov5s.yaml to train with pre-trained model ,in the course of train the model weight is 54.5MB,why it is so big than14.0MB?
![f276535546aadb36ec1f15382224c6c](https://user-images.githubusercontent.com/67963112/150910214-8f48cc53-59a6-45fa-a9d4-4a1d0c7842a4.jpg)

**Comment by glenn-jocher** [comment]
👋 Hello! Thanks for asking about **training checkpoints**.  YOLOv5 🚀 checkpoints should be about 4X the size of final trained checkpoints, as they carry not just a FP16 model, but a FP16 [EMA](https://www.tensorflow.org/api_docs/python/tf/train/ExponentialMovingAverage) and an FP32 optimizer of the same size as the model (each model parameter has its own FP32 gradient saved within the optimizer). Final checkpoints contain only an FP16 model, with the EMA and optimizer both stripped after the final epoch of training:
https://github.com/ultralytics/yolov5/blob/b4a29b5a8d63a8c2d4a8929942b44e8969c5dddd/train.py#L423-L425

The `strip_optimizer()` function updates the checkpoint dictionary value for `model`, replacing it with `ema`, and sets `ema` and `optimizer` keys to `None`, which will reduce the checkpoint size by 3/4.
https://github.com/ultralytics/yolov5/blob/b4a29b5a8d63a8c2d4a8929942b44e8969c5dddd/utils/general.py#L741-L755

Final trained checkpoints after `strip_optimizer()` should match the README table sizes below.
<p align="center"><img width="700" alt="YOLOv5 Models" src="https://github.com/ultralytics/yolov5/releases/download/v1.0/model_comparison.png"></p>

You can also run `strip_optimizer()` manually on any checkpoint to convert it into finalized weights ready to train a new model:
```python
from utils.general import strip_optimizer

strip_optimizer('path/to/best.pt')
!python train.py --weights path/to/best.pt  # use best.pt to train a new model
```

Good luck 🍀 and let us know if you have any other questions!

**Comment by jiangjiaxiaotianxia** [comment]
very thank you ,I know ,GOOD luckly
